### ISSUE TITLE
Confusion Matrix

### ISSUE THREAD

**Issue by Auth0rM0rgan** [kratzert/finetune_alexnet_with_tensorflow#45]
Hello,
 
Thanks for the code. Actually, This is not an issue, It's just a question :)

I want to know how the classification model is working which, I need confusion matrix in order to understand. Tensorflow has a function for it which is `(tf.confusion_matrix(labels,predictions,num_classes=None)` but I am confused about what should be the labels and predictions values?

Thanks.

**Comment by Auth0rM0rgan** [comment]
Find the solution to build confusion matrix
